configs/defs/generic_linux.textproto

```diff
@@ -6,26 +6,25 @@ benchmark_configs: {
   compliance_note: {
     version: { cpe_uri: "cpe:/o:centos:centos:7.%02" version: "3.1.2" benchmark_document: "CIS Centos Linux 7.x" }
     version: { cpe_uri: "cpe:/o:rockylinux:rockylinux:8.5" version: "1.0.0" benchmark_document: "CIS Rocky Linux 8" }
-    title: "Ensure mounting of cramfs filesystems is disabled"
+    title: "Ensure cramfs kernel module is not available"
```

Comment:
Is this new description from an updated benchmark doc? I see that CIS CentOS 7 version 4.0.0 has a new benchmark `1.1.1.1 Ensure cramfs kernel module is not available`.

However the CIS CentOS 7 v3.1.2 doc still has the old benchmark definition `Ensure mounting of cramfs filesystems is disabled`. So we should either
* Keep the old check as is and add a new check with doc version 4.0.0 specified in the version field, or
* Overwrite the check like you did here but also update all version strings from 3.1.2 to 4.0.0 (same for other OSes like rockylinux 8.5 1.0.0 -> 2.0.0)

Reply:
Actually we've no reason to maintain older CIS benchmark versions so I think it's fine to go with option 2.
Can you do the following:
* Remove the legacy cramfs-module-not-available check you added
* Bump our used benchmark for CentOS to 4.0.0: Update the benchmark document versions to 4.0.0 everywhere, in including other non-modified benchmark definitions, and the versions in the usage in configs/reduced/centos_7
* Do the same for Rocky 1.0.0 -> 2.0.0